### DISCUSSION TITLE
Ant Media Enterprise doesn't accept multiple stream ingestion

### DISCUSSION THREAD

**Discussion by elijahwere** [ant-media/Ant-Media-Server#6398] [FAQs]
Hello!

I have Ant Media Server set up with adent.io script. 
On my website that's running adent.io script, users create live streams using their web cams and then Ant Media is suspposed to capture the ingested streams. However, Ant Media only broadcasts one stream and not multiple streams based on the number of users who are broadcasting to it.

Do I need to add the `ingestingStreamLimit` to Ant Media?

**Comment by Mohit-3196** [comment]
Hi @elijahwere,
How are you?

As far as I remember, these scripts are for 1:N broadcast scenario and not N-N conferencing scenario.
If you share some ways in which I can test this, then I can check and suggest. Alternatively, you can ask Adent about it.

Thank you.
